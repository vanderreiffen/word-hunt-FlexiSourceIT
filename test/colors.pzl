UPVPHATNVSGYXQCRIKQM
XFWRXCDERUTGISLJXOXI
GRLOWWGTLWTQEVDUFCVM
UBNXTURSOBJIRDWLTKPX
XDEJGYELLOWRZEKKTVRF
YDMBLUEZGYUISJOGLPHF
AXVBHSNSEBJBCRZINTFL
WVCGBCAIJCHDGNDGREEN
YSHQGOYYVCTAGCVVDBYZ
CJMIPEFHVJHTWYOKLTZS
CVHCCSQQYMOFVWEWKRRU
OQQGLUHENDJIANIWHAVY
IIKJJGNQJBTTZXRTKFUT
LRNVWRDGDUWZYFUGEDKF
ZXGDESJUIUZMOCYAAJYC
HUEUTXUDIELLJYBKKMFS
FHGBRPDSBGHCTARMWRJZ
DJNKXJEDHPUAUUGNEAAD
APQWLKNLBZJFPMWLNAQA
PPANGSTFAOSHEMVCBKFP

BLUE
YELLOW
GREEN
RED