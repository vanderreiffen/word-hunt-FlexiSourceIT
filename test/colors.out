BLUE (4,6) (7,6)
YELLOW (6,5) (11,5)
GREEN (7,3) (7,7)
RED (9,2) (7,2)
